nc-news/components/Article.jsx

```diff
@@ -0,0 +1,33 @@
+import { useEffect, useState } from "react";
+import axios from "axios";
+import { Routes, Route, useParams } from "react-router-dom";
+
+const Article = () => {
+  const [article, setArticle] = useState({});
+
+  let { article_id } = useParams();
+
+  useEffect(() => {
+    axios
```

Comment:
I would strongly recommend extracting this api call into a separate function in a separate file. This way you can use the axios base url. [See these notes](https://notes.northcoders.com/courses/js-front-end/axios) 👍